acra/security-controls/masking/_index.md

```diff
@@ -2,3 +2,56 @@
 title: Masking
 bookCollapseSection: true
 ---
+
+# Masking
+
+Partial column encryption.
+Hides encrypted part from those who don't have right decryption keys.
+In case of decryption failure, encrypted data will be replaced with a placeholder.
```

Comment:
Well, you *did* say that it's short, but this is *way too* short of a description, in my opinion.

I believe you should go into more detail on what masking does and provide some use cases for which masking makes a good fit. Like, why would you even want to disclose anything to a user without the right decryption keys or authorization to access the data. (First of all, I don't think it's about whether *the user* has the keys or not, but rather if the AcraServer instance which the data is accessed through has the required keys.)

Reply:
Agree. The introduction should be expanded a bit. Following use-cases can be mentioned:
- credit-card number masking (when part of number is shown in banking);
- email/phone number masking (in password recovery).